source/code/plugin/health/health_container_cpu_memory_aggregator.rb

```diff
@@ -221,6 +223,42 @@ def get_records
                 container_cpu_memory_records.push(health_record)
             }
 
+            # If all records that were sent previously are present in current set, this will not be executed
+            if @@cpu_last_sent_monitors.keys.size != 0
```

Comment:
if there are any remaining monitors, then the workload has been deleted. So send a transition to NONE state for the monitor